examples/modals/persistent.py

```diff
@@ -0,0 +1,59 @@
+import nextcord
+
+TESTING_GUILD_ID = 123456798  # Replace with your testing guild id
+
+class Feedback(nextcord.ui.Modal):
+    def __init__(self):
+        super().__init__(
+            title="Feedback",
+            custom_id="feedback",
+            timeout=None,
+        )
+        self.discovered = nextcord.ui.TextInput(
+            label="How did you discover the bot?",
+            required=False,
+            style=nextcord.TextInputStyle.paragraph,
+            custom_id="discovered",
+        )
+        self.rating = nextcord.ui.TextInput(
+            label="How would you rate the bot?",
+            placeholder="I would give the bot a ten out of ten...",
+            custom_id="rating",
+        )
+        self.improve = nextcord.ui.TextInput(
+            label="How could the bot improve?",
+            style=nextcord.TextInputStyle.paragraph,
+            required=False,
+            custom_id="improve",
+        )
+        self.add_item(self.discovered)
+        self.add_item(self.rating)
+        self.add_item(self.improve)
+    
+    async def callback(self, interaction: nextcord.Interaction):
+        await interaction.send(
+            f"{interaction.user.mention} feedback :\n"
+            f"Rating: {self.rating.value}\n"
+            f"Where did he discover the bot: {self.discovered.value}\n"
+            f"How could the bot improve: {self.improve.value}\n"
+        )
+
+bot = nextcord.Client()
+
+feedback_modal = None
+
+@bot.event
+async def on_ready():
+    global feedback_modal
```

Comment:
Is there not a way to do it without a global variable?

Reply:
We can use a cog, but this is a too advanced concept I think

Reply:
Could you use a bot attribute or something

Reply:
Something like `bot.feedback_modal` ?